pkg/network/usm/usm_http2_monitor_test.go

```diff
@@ -506,12 +506,13 @@ func (s *usmHTTP2Suite) TestRawTraffic() {
 			// a single program.
 			messageBuilder: func() [][]byte {
 				const settingsFramesCount = 238
+				const endStream = true
```

Comment:
make it a global const, no much point to have it defined multiple times

Reply:
I'm confused
The original intention was to control endHeaders, but the change in the file is only for end stream